app/models/mixins/supports_feature_mixin.rb

```diff
@@ -73,6 +73,7 @@ module SupportsFeatureMixin
     :backup_restore                      => 'CloudVolume backup restore',
     :cinder_service                      => 'Cinder storage service',
     :cinder_volume_types                 => 'Cinder volume types',
+    :conversion_host                     => 'Conversion host support',
```

Comment:
Would changing the description to something like "Conversion host capable" be helpful?

Reply:
@jameswnl Done!